### PR TITLE
Add the endpoints command example

### PR DESCRIPTION
**Pull request by fisherxu**
Now we have added the endpoints proxy-config subcommand, so add this ops example.
xref: [#8512](https://github.com/istio/istio/pull/8512)

### CODE REVIEW

File: content/help/ops/traffic-management/proxy-cmd/index.md [hunk 1]
```diff
@@ -278,6 +278,70 @@ one route that matches on everything. This route tells Envoy to send the request
         }
     ]
     {{< /text >}}
+    
+1. When envoy use the `serviceName` field as a key that have got the list of Endpoints and then will proxy the request to one of them.
```
Comment (liamawhite) [inline content/help/ops/traffic-management/proxy-cmd/index.md]: To see the endpoints currently available for the cluster use the `proxy-config endpoints` command.

File: content/help/ops/traffic-management/proxy-cmd/index.md [hunk 2]
```diff
@@ -278,6 +278,70 @@ one route that matches on everything. This route tells Envoy to send the request
...
+
+    {{< text bash json >}}
+    $ istioctl proxy-config endpoints productpage-v1-6c886ff494-7vxhs --cluster="outbound|9080||reviews.default.svc.cluster.local" -o json
```
Comment (liamawhite) [inline content/help/ops/traffic-management/proxy-cmd/index.md]: --cluster outbound|9080||reviews.default.svc.cluster.local

Comment (liamawhite) [reply content/help/ops/traffic-management/proxy-cmd/index.md]: and could you use the summary here not `-o json`

File: content/help/ops/traffic-management/proxy-cmd/index.md (hunk 1 above)
Comment (fisherxu) [reply content/help/ops/traffic-management/proxy-cmd/index.md]: Done.

File: content/help/ops/traffic-management/proxy-cmd/index.md (hunk 2 above)
Comment (fisherxu) [reply content/help/ops/traffic-management/proxy-cmd/index.md]: Done.

**Comment by fisherxu** [comment]
/hold
waiting for [#8569](https://github.com/istio/istio/pull/8569)

File: content/help/ops/traffic-management/proxy-cmd/index.md [hunk 3]
```diff
@@ -278,6 +278,16 @@ one route that matches on everything. This route tells Envoy to send the request
         }
     ]
     {{< /text >}}
+    
+1. To see the endpoints currently available for the cluster use the proxy-config endpoints command.
+
+    {{< text bash json >}}
+    $ istioctl proxy-config endpoints productpage-v1-6c886ff494-7vxhs --cluster=outbound|9080||reviews.default.svc.cluster.local
```
Comment (liamawhite) [inline content/help/ops/traffic-management/proxy-cmd/index.md]: Nit: Could you remove the `=` here

File: content/help/ops/traffic-management/proxy-cmd/index.md [hunk 4]
```diff
@@ -278,6 +278,16 @@ one route that matches on everything. This route tells Envoy to send the request
         }
     ]
     {{< /text >}}
+    
+1. To see the endpoints currently available for the cluster use the proxy-config endpoints command.
```
Comment (liamawhite) [inline content/help/ops/traffic-management/proxy-cmd/index.md]: Self-nit: To see the endpoints currently available for *this* cluster use the proxy-config endpoints command.

File: content/help/ops/traffic-management/proxy-cmd/index.md (hunk 3 above)
Comment (fisherxu) [reply content/help/ops/traffic-management/proxy-cmd/index.md]: Done. But wondering what's difference about have or no `=`, all can work.

Comment (liamawhite) [reply content/help/ops/traffic-management/proxy-cmd/index.md]: Just consistency with the rest of the examples

Comment (fisherxu) [reply content/help/ops/traffic-management/proxy-cmd/index.md]: ok :-)

**Comment by geeknoid** [comment]
There are a few CI failures to address. Several of them will go away by rebasing your repo. But there remain a few you'll need to take care of explicitly.

**Comment by geeknoid** [comment]
Put proxy-config between backticks to fix the last CI problem,

**Comment by fisherxu** [comment]
@geeknoid @sdake Thanks for merged :) and Ci also all green now

**Comment by andraxylia** [comment]
@geeknoid is master the current istio.io ?

We do not see the changes in this PR on the current website, and they are very useful.

**Comment by geeknoid** [comment]
istio.io reflects what is in the release-1.0 branch of this repo.
preliminary.istio.io reflects what is in the master branch of this repo.

When 1.1 comes out, we will branch master to release-1.1 and point istio.io to release-1.1.
Until then, if you want changes to be reflected immediately on istio.io, you need to apply the changes to the release-1.0 branch.

**Pull request merged**